packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -84,6 +84,8 @@ const chatMessageStyles: ComponentSlotStylesPrepared<
 
   actionMenu: ({ props: p, variables: v }): ICSSInJSStyle => ({
     backgroundColor: v.backgroundColor,
+    border: '1px solid',
+    borderColor: v.reactionGroupBorder,
```

Comment:
It should be:

```suggestion
    borderColor: v.reactionGroupBorderColor,
```

Or

```suggestion
    border: v.reactionGroupBorder,
```